addons/website_livechat/static/src/web/persona_model_patch.js

```diff
@@ -0,0 +1,21 @@
+import { Persona } from "@mail/core/common/persona_model";
+
+import { patch } from "@web/core/utils/patch";
+import { deserializeDateTime } from "@web/core/l10n/dates";
+
+const { DateTime } = luxon;
+
+patch(Persona.prototype, {
+    get historyString() {
+        if (!this.history_data || this.history_data.length === 0) {
+            return "";
+        }
+        const history = [];
+        for (const h of this.history_data) {
+            const [label, date] = h;
+            const time = deserializeDateTime(date).toLocaleString(DateTime.TIME_24_SIMPLE);
+            history.push(`${label} (${time})`);
+        }
+        return history.join(" → ");
+    },
```

Comment:
:pick: easier to follow?

```suggestion
    get historyString() {
        const history = [];
        for (const h of this.history_data ?? []) {
            const [label, date] = h;
            const time = deserializeDateTime(date).toLocaleString(DateTime.TIME_24_SIMPLE);
            history.push(`${label} (${time})`);
        }
        return history.join(" → ");
    },
```